contrib/builder/deb/armhf/ubuntu-xenial/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM armhf/ubuntu:xenial
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev libsqlite3-dev pkg-config libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.7.3
+RUN curl -fSL "https://storage.googleapis.com/golang/go${GO_VERSION}.linux-armv6l.tar.gz" | tar xzC /usr/local
```

Comment:
We should start working on an update/create a generate.sh for armhf, like we have for amd64; https://github.com/docker/docker/blob/master/contrib/builder/deb/amd64/build.sh